packages/flutter/lib/src/widgets/animated_child_switcher.dart

```diff
@@ -62,8 +158,58 @@ class AnimatedChildSwitcher extends StatefulWidget {
   /// The duration over which to perform the cross fade using [FadeTransition].
   final Duration duration;
 
+  /// The builder function used to generate custom transitions for
+  /// [AnimatedChildSwitcher].
+  ///
+  /// The default is [AnimatedChildSwitcher.defaultTransitionBuilder].
+  ///
+  /// See also:
+  ///   * [AnimatedChildSwitcherTransitionBuilder] for more information about
```

Comment:
You need a blank line before the list.
Also, only indent the bullet by one, so that the text is indented by 3.

Reply:
Fixed, and below.